google-cloud-compat-checker/src/main/java/com/google/cloud/compatchecker/GoogleCloudCompatChecker.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.compatchecker;
+
+import com.google.common.collect.ImmutableSet;
+import com.google.common.collect.Lists;
+import java.util.List;
+import java.util.Properties;
+import java.util.Set;
+import kr.motd.maven.os.Detector;
+
+public class GoogleCloudCompatChecker {
+  private static Set<String> supportedClassifiers =
+      ImmutableSet.of("linux-x86_64", "osx-x86_64", "windows-x86_64");
+
+  private static class OsDetector extends Detector {
+    public void detect(Properties props, List<String> classifierWithLikes) {
+      super.detect(props, classifierWithLikes);
+    }
+
+    @Override
+    protected void log(String s) {
+      // meh
```

Comment:
Yeah, the implementation inside the plugin just has log.info lines in its implementation.